views/outcome.erb

```diff
@@ -0,0 +1,3 @@
+<h1><%= @player_name%> picked <%=@player_choice%>!</h1>
+<h1>CPU picked <%= $computer_choice%>!</h1>
+<h1><%=@winner%> is the winner!</h1>
```

Comment:
Where is this @winner variable? I might be missing it somewhere, but in your app.rb you define the winner as $winner. Is that part of the issue of it not printing the name?